### DISCUSSION TITLE
print() showing in the <div> output instead of the console?

### DISCUSSION THREAD

**Discussion by ThomRoy** [pyscript/pyscript#1421] [Q&A]
Hello!
Is it possible to have a "print()" displayed in the `<div>` that is supposed to display the output of the code contained in `<py-repl>`, rather than in the console?
Thanks in d’avance!

**Comment by ThomRoy** [comment]
(For the record, I'm a teacher, and I'm trying to use PYSCRIPT to get my students to work on python code. You can see what I'm trying to do here: https://physique-pcsi.prepa-balzac.fr/approches-numeriques-python/equations-differentielles-avec-python/   - just type anything at the javascript prompt that comes up, it doesn't matter).

**Reply by patricklok** [reply]
the link is scam

**Reply by WebReflection** [reply]
it's not? ... it prompts your name then it asks to solve problems in Python? why would you say that?

**Comment by JeffersGlass** [comment]
This is a feature that's been merged a few weeks ago (#1106) but as we haven't had a release since then, it's not not part of a published release. 

Once that's part of a release, `print()` statements inside the REPL will show up "underneath" the REPL as well as in the \<py-terminal\>. Alternatively, an `output` attribute will be added to the \<py-repl\>, in which case output will be sent to the HTML element with a matching ID.

**Reply by ThomRoy** [reply]
Very nice, thanks a lot!!!
Any idea when this release is likely to roll up?

**Reply by henrikenquist** [reply]
Yeah, I'm also interested in the release date of this feature since I would like to use it in "production"...
(https://enquist.pythonanywhere.com/)
